### PR TITLE
BaSP-TG-76: Time sheet redux implementation

### PR DESCRIPTION
**Pull request by Agustin-Chazarreta**

### CODE REVIEW

File: src/Components/TimeSheets/index.jsx [hunk 1]
```diff
@@ -5,20 +5,23 @@ import Button from '../Shared/Buttons/buttons';
...
-      .then((response) => response.json())
-      .then((response) => setList(response.data));
-  };
+
+  const dispatch = useDispatch();
+  const list = useSelector((state) => state.timeSheets.timeSheetsList);
+  const isLoading = useSelector((state) => state.timeSheets.isLoading);
+  console.log(isLoading);
   useEffect(() => {
     fetch(`${process.env.REACT_APP_API_URL}/projects/`)
```
Comment (cellamarematias) [inline src/Components/TimeSheets/index.jsx:26]: Hi, this fetch should be in thunk.js.

File: src/Components/TimeSheets/index.jsx [hunk 2]
```diff
@@ -47,83 +44,89 @@ const TimeSheets = () => {
...
-            <Button
-              callback={() => {
-                setIsModalDelete(false);
-              }}
-              text={'Cancel'}
-            />
+        <ModalAddTimeSheet
+          isModalAdd={isModalAdd}
+          setIsModalAdd={setIsModalAdd}
+          fetchTimeSheets={() => getTimeSheets()(dispatch)}
```
Comment (cellamarematias) [inline src/Components/TimeSheets/index.jsx:89]: Hi, this fetch should be in thunk.js.

**Review by cellamarematias — CHANGES_REQUESTED** [review]

File: src/Components/TimeSheets/index.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Agustin-Chazarreta) [reply src/Components/TimeSheets/index.jsx:26]: to do this i need all the thunks of the others entities, it just a patch implementation.

[the remark this answers is not in the document]
Comment (cellamarematias) [reply src/Components/TimeSheets/index.jsx:26]: I get it!

**Review by cellamarematias — APPROVED** [review]

**Review by AnaMurinigo — APPROVED** [review]
Test Result: Pass

**Pull request merged**